.buildkite/integration.pipeline.yml

```diff
@@ -95,7 +95,6 @@ steps:
     command: |
       buildkite-agent artifact download build/distributions/** . --step 'package-it'
       .buildkite/scripts/steps/beats_tests.sh
-    # if: "build.env('CRON') == 'yes'"
```

Comment:
Let's uncomment this line

Reply:
Agent team, can you confirm the behavior you want? Currently the serverless beats tests are running with every single build, and it's been like that for 4 months. Do you only want the serverless beats tests running once per day?

Reply:
I found the history. It looks like @pchila intentionally enabled them to run with every build - https://github.com/elastic/elastic-agent/pull/4501

Reply:
Sorry, just found this PR via its backport but I can still add a bit of context:

This was supposed to be temporary to make sure that agent/beats changes on PRs would not break serverless beats tests.

The whole step was supposed to be moved to the beats CI pipeline at some point but here it still is 6 months ago.

@pierrehilbert @jlind23 do we have an idea when these tests can be moved off the agent pipeline and into beats CI?

Reply:
We were "only" waiting for buildkite to be available in the beats repository so I think we should be able to do it by now.

Reply:
Tracking with https://github.com/elastic/ingest-dev/issues/4128, currently set for sprint 37